app/policies/ingredient_category_policy.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class IngredientCategoryPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      if user.admin?
+        scope.all
+      else
+        false
+      end
+    end
```

Comment:
Remember using guard condition, check how cool it would look! (and always leave that empty space after the guard clause)
```ruby
def resolve
  return false unless user.admin?    

  scope.all
end
```